elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -111,17 +111,23 @@ class PfeContentSet extends PFElement {
     return this.parentNode ? this.parentNode.offsetWidth > breakpointValue : window.outerWidth > breakpointValue;
```

Comment:
TODO: What happens if I put something not pfe-tab or pfe-accordion in the rendered-component slot?